customRoutes/publishcve.js

```
@@ -0,0 +1,168 @@
+    // This isn't implemented yet
+    //
+    if (conf.pmcstrustedascna.includes("*")) {
+        if (conf.pmcstrustedascna.includes("-"+specificpmc)) {
+            return false;
+        }        
+        return true;
+    }    
+    if (conf.pmcstrustedascna.includes(specificpmc)) {
+        return true;
+    }
+    return false;
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement, ...

... this statement, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)